docs/gettingstarted/installation-macos.rst

```diff
@@ -18,7 +18,18 @@ PostgreSQL
 
 Saleor needs PostgreSQL version 9.4 or above to work. Get the macOS installer from the `PostgreSQL download page <https://www.postgresql.org/download/macosx/>`_.
 
-Make sure you keep track of the password you set for the administration account during installation.
+Then, create a new database and user. Make sure you keep track of the password you set for the administration account during installation. 
+
+.. code-block:: console
+
+ $ psql postgres
+
+and from the Postgres prompt:
+
+.. code-block:: psql
+
+ # CREATE ROLE saleor WITH LOGIN PASSWORD 'saleor' SUPERUSER;
+ # CREATE DATABASE saleor;
```

Comment:
You should not run those commands as root but as `postgres` user. Or it doesn't work that way on MacOS?

Reply:
On macOS you don't need a special system user.

The default prompt after issuing `psql postgres` is `postgres=#` though so I'm not sure `=>` is a good prompt.

Reply:
Okay then!